opencda/core/ml_libs/rl/rl_models/model.py

```diff
@@ -0,0 +1,395 @@
+import torch
+import torch.nn as nn
+from typing import List, Tuple, Union, Dict, Optional
+
+from opencda.core.ml_libs.rl.utils.img_encoder import BEVSpeedConvEncoder
+from ding.model.common.head import DuelingHead, RegressionHead, ReparameterizationHead, MultiHead, DiscreteHead
+
+
+class DQNRLModel(nn.Module):
```

Comment:
Add some docstrings for this class for better demonstration of the functionality of this class~